concepticondata/references/references.bib

```diff
@@ -3102,3 +3102,78 @@ @misc{Hsiu2015
   doi = {10.5281/zenodo.1127804},
   publisher={Center for Research in Computational Linguistics}
 }
+
+@article{Maciejewski2016,
+  title={{Relative meaning frequencies for 100 homonyms: British eDom norms}},
+  author={Maciejewski, Greg and Klepousniotou, Ekaterini},
+  journal={Journal of Open Psychology Data},
+  volume={4},
+  year={2016},
+  publisher={Ubiquity Press}
+}
+
+@misc{Parks1998,
```

Comment:
try to avoid `@misc` entries in bibliographies. It is better to just label them as a `@book`. And if you have a publisher, you also need an `address`.